src/template/Header/Header.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react'
+import { connect } from 'react-redux'
+
+import Preloader from 'ui-kit/preloader/preloader'
+
+import { requestImages } from '../../redux/action'
+
+import '../scss/Hedaer.scss'
+
+const Header = props => {
+  const { onImgFetch, isLoading } = props
+  useEffect(() => {
+    onImgFetch()
+  }, [onImgFetch])
+
+  if (isLoading) {
+    return <Preloader />
+  }
+  const image = props.imageDaily.image
```

Comment:
please have a look at object destructuring:
this:

```
const Header = props => {
  const image = props.imageDaily.image
}
```

is equivalent to this:
```
const Header = ({ imageDaily })=> {
  const { image } = imageDaily
}
```

Reply:
the second is better and more readable